style.css

```diff
@@ -0,0 +1,181 @@
+:root{
+    --white: #FFF;
+    --blue: #199BE2;
+    --light-blue: #50E6FF;
+    --dark-blue: #0D62AB;
+    --light-blue-border: #35C1F1;
+    --dark-blue-border: #0078D4;
+    --grey: #888;
+    --light-grey: #EFEFEF;
+    --shadow: #00000020;
+    --body-background: #F5F5F7;
+}
+
+body{
+    margin: 0;
+    padding: 0;
+    background-color: var(--body-background);
+}
+
+header{
+    padding: 15px 30px;
+    background-color: var(--white);
+    box-shadow: 0 3px 10px var(--shadow);
+    position: sticky;
+    top: 0;
+}
+
+header a{
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    color: var(--blue);
+    width: max-content;
+    font-family: 'Cabin';
+}
+
+header h1{
+    margin: 0;
+    margin-left: 10px;
+}
+
+.tool__bar{
+    text-align: center;
+    margin: 15px 0;
+    font-family: 'Cabin';
+}
+
+.tool__bar > div:first-child{
+    margin-bottom: 10px;
+}
+
+input{
+    border: none;
+    font-size: 19px;
+    border-bottom: 1px solid var(--blue);
+    background-color: var(--body-background);
+}
+
+button{
+    border: none;
+    outline: none;
+    cursor: pointer;
+    padding: 5px 10px;
+    border-radius: 5px;
+    font-family: 'Cabin';
+    color: var(--dark-blue);
+}
+
+#sort__details{
+    font-size: 17px;
+}
+
+.sort__btn{
+    margin: 5px;
+    font-size: 17px;
+}
+
+.sort-button--active{
+    background-color: var(--light-blue);
+}
+
+.cur__sort{
+    background-color: var(--light-blue) !important;
+}
+
+.repos__container{
+    width: max-content;
+    margin: 0 auto;
+    padding: 10px;
+    box-sizing: border-box;
+    font-family: 'Cabin';
+    color: var(--grey);
+}
+
+.repo__card{
+    overflow-y: hidden;
+    max-height: 300px;
+    width: 390px;
+    margin-bottom: 10px;
+    padding: 0 10px;
+    transition: max-height 0.35s ease-in-out;
+    display: flex;
+    align-items: flex-start;
+    justify-content: space-between;
+    background-color: var(--white);
+    border-radius: 10px;
+}
+
+.repo__card > *{
+    margin: 25px 0;
+}
+
+.repo__card > div{
+    width: 100%;
+    margin: 25px 10px;
+}
+
+.repo__card > div > a{
+    font-size: 19px;
+    text-decoration: none;
+    color: var(--blue);
+    font-family: 'Cabin';
+    word-break: break-all;
+}
+
+.repo__card > div > div{
+    margin-top: 10px;
+    display: flex;
+}
+
+.repo__card > div > div > div{
```

Comment:
Try to avoid a chain of selectors like this. It's best to follow [BEM methodology](https://en.bem.info/methodology/) for writing maintainable CSS (but first you should understand the concept of CSS specificity).